src/gsy_e/models/leaves.py

```diff
@@ -76,6 +77,7 @@ class LeafBase:
     instantiated and serialized in a more compact format
     """
     strategy_type = None
+    strategy = None
```

Comment:
Can you please explain why this is needed?

Reply:
In order to make pylint happy.